app/src/main/java/vn/sunasterisk/music_70/data/remote/RemoteDataSource.kt

```diff
@@ -0,0 +1,24 @@
+package vn.sunasterisk.music_70.data.remote
+
+class RemoteDataSource : TrackDataSource.Remote {
```

Comment:
Nếu không cần truyền tham số thì chỉ cần như này là đủ nhé 
```suggestion
object RemoteDataSource : TrackDataSource.Remote {
```